configuration/device-tree.md

```diff
@@ -595,17 +660,19 @@ Two points to note:
 
 2. Only overlays and parameters applied at run-time can be unloaded - an overlay or parameter applied by the firmware becomes "baked in" such that it won't be listed by `dtoverlay` and can't be removed.
 
-<a name="part3.5.2"></a>
 #### 3.5.2 The dtparam command
+<a name="part3.5.2"></a>
 
-`dtparam` creates an overlay that has the same effect as using a dtparam directive in `config.txt`. In usage it is largely equivalent to `dtoverlay` with an overlay name of `-`, but there are a few small differences:
+`dtparam` creates an overlay that has largely the same effect as using a dtparam directive in `config.txt`. In usage it is largely equivalent to `dtoverlay` with an overlay name of `-`, but there are a few differences:
 
 1. `dtparam` will list the help information for all known parameters of the base DTB. Help on the dtparam command is still available using `dtparam -h`.
 
 2. When indicating a parameter for removal, only index numbers can be used (not names).
 
-<a name="part3.5.3"></a>
+3. Not all Linux subsystems respond to the addition of devices at runtime - I2C, SPI and sound devices work, but some won't.
```

Comment:
Does this imply that the "`dtparam` creates an overlay..." a few lines earlier should be tweaked to "`dtparam` creates and loads an overlay..." or similar?

Reply:
Yes - "creates and loads".